Profile ui #5

JamesClark123 opened the pull request.

JamesClark123 committed:
Adds the base UI layout for the profile page

Planning on splitting out the different components into seperate files on future commits - this commit also adds a dependency to @material-ui/icons

JamesClark123 committed:
Adds some style updates and removes unused packages

skassam21 commented on client/src/pages/Profile.js:
```diff
@@ -0,0 +1,316 @@
+  }
+  return (
+    <div className={classes.backgroundColor}>
+      <AppBar className={classes.topBar}>
```
Did you see my request in the other PR - it would be great to put this actually at a higher level component (in the App, and make this a component itself) so it could exist in all the pages easily

skassam21 reviewed (commented):
Do the fonts render properly? I remember commenting about that in the other PR, and wanted to make sure they are addressed

skassam21 commented:
Are you using a prettier.io or a linter btw on the front-end?

JamesClark123 commented:
@skassam21  I'm using prettier.io on VSCode (I think that's what you recommended?)

JamesClark123 committed:
Moves header, fixes font rendering

Moves header to it's own component, renders in App.js - conditionally depending on the page. Fixes font's not rendering

JamesClark123 commented:
@skassam21  @tinla94 this fixes the issues, may I merge now?

aeciorc commented on client/src/pages/Profile.js:
```diff
@@ -0,0 +1,224 @@
+import React, { useState } from "react";
```
I realize this has been merged already, but I feel like this component could use a bit of refactoring. The css got a little to big and it would be nice to move it to a separate file. I also think the lists could be their own component.
